changelog

```diff
@@ -0,0 +1,34 @@
+* Fri Oct 02 2015 Francisco Romero <francisco.romerobueno@telefonica.com> 0.1.0
```

Comment:
I understand this file (created in this first release) will be the cumulative changelog between versions (the one that is typically included in the RPM .spec in other projects). Is this correct?